feat: add beagle flutter documentation (#717)
ZupIT/beagle-docs: content/en/resources/customization/beagle-for-flutter/customization/_index.md

---
title: Customization
weight: 6
description: >-

>   You will find here information of how to do customizations in Beagle Flutter.

Reviewer: ```suggestion
 In this section, you will find information on how to do customizations in Beagle Flutter.
```
